File: Python/exercicio01.py
# Faça um algoritimo para somar dois numeros e multiplicar o resultado pelo primeiro numero

n1 = int(input("Informe o primeiro numero: "))
n2 = int(input("Informe o segundo numero: "))

soma = n1 + n2
multiplicacao = soma * n1

print("A soma de {} + {} = {}".format(n1, n2, soma))
print("A multiplicação de {} * {} = {}".format(soma, n1, multiplicacao))

File: C/Exercicio01.c
//Faça um algoritimo para somar dois numeros e multiplicar o resultado pelo primeiro numero

#include<stdio.h>

int main(){
    int n1, n2, soma, multiplicar;

    printf("Informe o primeiro numero: ");
    scanf("%d", &n1);
    
    printf("Informe o segundo numero: ");
    scanf("%d", &n2);
    
    soma = n1 + n2;
    multiplicar = soma * n1;
    
    printf("A soma de %d + %d = %d\n", n1, n2, soma);
	printf("A multiplicacao de %d + %d = %d", soma, n1, multiplicar);    
}
